css/event-detail.css

```diff
@@ -0,0 +1,84 @@
+html {
+	box-sizing: border-box;
+}
+
+.template-wrapper {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	padding-left: 20px;
+	padding-right: 20px;
+	width: var(--mobile-wrapper-width);
+	margin: 0 auto;
+	max-width: var(--wrapper-width-max);
+}
+
+.template-wrapper img {
+	align-items: center;
+}
+
+.template-wrapper h4 {
+	text-align: center;
+	color: var(--color-onyx);
+}
+
+.template-wrapper p {
+	color: var(--color-onyx);
+	align-items: center;
+}
+
+/* Needs to be centered, and button 3 attributes */
+.template-button {
+	width: var(--mobile-wrapper-width);
+	max-width: var(--wrapper-width-max);
+	min-height: 50px;
+	background-color: var(--color-amaranth);
+	color: var(--color-white);
+	border: none;
+	border-radius: 10px;
```

Comment:
Please use variables

Reply:
this is in another file that has now been removed